remote_attestation/java/src/com/google/oak/remote_attestation/crypto/AeadEncryptor.java

```diff
@@ -67,40 +68,40 @@ public AeadEncryptor(byte[] key) {
         // single encrypted field.
         byte[] encryptedData = encryptor.doFinal(data);
 
-        // Add `initializationVector` as a prefix to the `encryptedData`.
-        return Bytes.concat(initializationVector, encryptedData);
+        // Add `nonce` as a prefix to the `encryptedData`.
```

Comment:
nit: This comment is no longer accurate, as it is not a prefix but part of the object.